tests/integration/conftest.py

```diff
@@ -204,25 +204,30 @@ def pdf_entity_data_row(client):
     }
 
 
-@pytest.fixture(scope="session")
-def conversation_entity_data_row(client):
-    conversation_url = client.upload_file('tests/assets/conversation-1.json')
-
-    return {"row_data": conversation_url, "global_key": str(uuid.uuid1())}
+@pytest.fixture()
+def conversation_entity_data_row(client, rand_gen):
+    return {
+        "row_data":
+            "https://storage.googleapis.com/labelbox-developer-testing-assets/conversational_text/1000-conversations/conversation-1.json",
+        "global_key":
+            f"https://storage.googleapis.com/labelbox-developer-testing-assets/conversational_text/1000-conversations/conversation-1.json-{rand_gen(str)}",
+    }
 
 
 @pytest.fixture
 def project(client, rand_gen):
     project = client.create_project(name=rand_gen(str),
-                                    queue_mode=QueueMode.Dataset)
+                                    queue_mode=QueueMode.Batch,
+                                    media_type=MediaType.Image)
     yield project
     project.delete()
 
 
 @pytest.fixture
 def batch_project(client, rand_gen):
```

Comment:
this should be removed now and all references should be replaced to use `project` (which is the same fixture now)